src/proxy.js

```diff
@@ -262,6 +272,91 @@ function iterateMapValues(state, prop, receiver) {
     }
 }
 
+/**
+ * Set drafts
+ */
+
+const setGetters = {
+    [DRAFT_STATE]: state => state,
+    size: state => source(state).size,
```

Comment:
Looks like `mapGetters` and `setGetters` share some values.
```js
const setGetters = {
  ...mapGetters,
```
You can omit `[DRAFT_STATE]`, `size`, and `has` here.

Reply:
This would add other map getters (like `set`), which we don't have overrides for, to set getters (Set do not have `set` method). Anyway, it's only 3 of them and they are really small. If you're firm on this we certainly can figure out some way to share this code, but it's just a couple of lines, so I'm not sure it's worth it.

Reply:
Good point. We can leave it as-is